DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -1,66 +1,117 @@
 ﻿namespace DigitalLearningSolutions.Data.Tests.Services
 {
+    using System.Collections.Generic;
     using System.Linq;
+    using DigitalLearningSolutions.Data.Models.User;
     using DigitalLearningSolutions.Data.Services;
     using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
     using FluentAssertions;
     using NUnit.Framework;
 
     public class UserServiceTests
     {
         private IUserService userService;
+        private IUserDataService userDataService;
 
         [SetUp]
         public void Setup()
         {
-            var connection = ServiceTestHelper.GetDatabaseConnection();
-            userService = new UserService(connection);
+            userDataService = A.Fake<IUserDataService>();
+            userService = new UserService(userDataService);
         }
 
         [Test]
-        public void GetAdminUserByUsername_Returns_admin_user()
+        public void GetUsersByUsername_Returns_admin_user_and_delegate_users()
         {
             // Given
             var expectedAdminUser = UserTestHelper.GetDefaultAdminUser();
+            var expectedDelegateUsers = UserTestHelper.GetDefaultDelegateUser();
+            A.CallTo(() => userDataService.GetAdminUserByUsername(A<string>._)).Returns(UserTestHelper.GetDefaultAdminUser());
```

Comment:
Should these return "expected admin user" etc (across all the tests)?

Reply:
I think that they can be, as these should be testing what the UserService functions return rather than what is passed in to the UserService function, so it should be safe to reuse the expected variable. So I think that might be a better way forward.